aioshelly/rpc_device/device.py

```diff
@@ -162,6 +165,8 @@ async def initialize(self, async_init: bool = False) -> None:
             async with async_timeout.timeout(DEVICE_IO_TIMEOUT):
                 await self._wsrpc.connect(self.aiohttp_session)
                 await self.update_config()
+                await self.update_device_info()
+                await self.update_profiles()
```

Comment:
As noted in my previous comment, I prefer the user of the lib to do it when needed, by calling `get_profiles` and not storing the data here. This is not supported for most of the devices and sleeping devices might have problems with this additional call.

Reply:
make update_profiles to query api only if "profile" is defined in `/shelly`